sasmodels_fit.py

```diff
@@ -0,0 +1,43 @@
+from sasmodels.core import load_model
+from sasmodels.bumps_model import Model, Experiment
+from sasdata.dataloader.loader import Loader
+
+from bumps.names import *
+from bumps.fitters import fit
+#from bumps.formatnum import format_uncertainty
+
+import numpy as np
+
+
+loader = Loader()
```

Comment:
I think we should use `pytest` to write multiple tests, and have proper control.
We should put the tests in a folder at the top level named `tests`.